executor/errors.go

```diff
@@ -26,10 +26,12 @@ const (
 	codeResultIsEmpty
 	codeErrBuildExec
 	codeBatchInsertFail
+	codeGetStartTS
 )
 
 // Error instances.
 var (
+	ErrGetStartTS      = terror.ClassExecutor.New(codeGetStartTS, "Get Start TS")
```

Comment:
`Get  Start TS` sounds like an action, not an error.

Reply:
So, any suggestion ?